helpers/baseMap.js

```diff
@@ -45,14 +46,42 @@ async function getBasemap(id, validFrom) {
       version = document.versions.shift();
     }
 
-    if (version.data) {
-      const response = await fetch(version.data);
-      if (response.ok) {
-        return await response.json();
-      } else {
-        return undefined;
+    if (isWide || !version.mobile) {
+      retVal.data = await fetchJSON(version.data);
+    } else {
+      retVal.mobile = [];
+      retVal.data = await fetchJSON(version.mobile.shift().data);
+      for (const baseMap of version.mobile) {
+        retVal.mobile.push({ data: await fetchJSON(baseMap.data) });
+      }
+    }
+    
+    if (version.miniMaps) {
+      retVal.miniMaps = [];
+      for (const miniMap of version.miniMaps) {
+        if (isWide && miniMap.type === "contentWidth" || !isWide && miniMap.type === "mobile") {
+          let miniMapCopy = { ...miniMap };
+          miniMapCopy.data = await fetchJSON(miniMap.data);
+          retVal.miniMaps.push(miniMapCopy);
+        }
       }
     }
+
+    return retVal;
+  } catch (error) {
+    return undefined;
+  }
+}
+
+async function fetchJSON(url) {
+  if (!url) return undefined;
```

Comment:
I think, this isn't necessary. A undefined url will be response.ok false or the try...catch block returns undefined